certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
+
+CAs SHALL NOT issue Certificates containing any Fully Qualified Domain Names that are not under the gTLDs for DotGov or DotMil.  
 
-Within 30 days after ICANN has approved a new gTLD for operation, as evidenced by publication of a contract with the gTLD operator on [www.ICANN.org] each CA MUST (1) compare the new gTLD against the CA's records of valid certificates and (2) cease issuing Certificates containing a Domain Name that includes the new gTLD until after the CA has first verified the Subscriber's control over or exclusive right to use the Domain Name in accordance with Section 3.2.2.4.
+Approval of certificate applications requires successful completion of validation per Section 3.2. 
+
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CAs SHALL use this information to identify subsequent suspicious certificate requests and MAY use it as the basis for rejecting a certificate request.
 
-Within 120 days after the publication of a contract for a new gTLD is published on [www.icann.org], CAs MUST revoke each Certificate containing a Domain Name that includes the new gTLD unless the Subscriber is either the Domain Name Registrant or can demonstrate control over the Domain Name.
 
 ### 4.2.3 Time to process certificate applications
-Certificate applications SHALL be processed and a certificate issued within 30 days of identity verification.
+Certificate applications SHALL be processed and a certificate issued within 30 days of the first identity verification.
 
 ## 4.3 Certificate issuance
 
 ### 4.3.1 CA actions during certificate issuance
-Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
+Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.  Issuance of a CA certificate by the Root CA SHALL require written authorization by the Policy Authority.  
+
+All end entity certificates for Domain Validation SSL and Organizational Validation SSL SHALL assert a Certificate Transparency (CT) Signed Certificate Timestamp (SCT) via the x509v3 certificate extension.  The Issuing CA SHALL submit a precertificate to a minimum of two Certificate Transparency Logs.  Information included in the end entity certificates SHALL NOT be redacted prior to submission to the Certificate Transparency Logs.  At a minimum, the Issuing CA SHALL request and include two SCTs in the x509v3 certificate extension:
```

Comment:
No, that's not current policy. Chrome [defines "CT Qualified"](https://a77db9aa-a-7b23c8ea-s-sites.googlegroups.com/a/chromium.org/dev/Home/chromium-security/root-ca-policy/CTPolicyMay2016edition.pdf?attachauth=ANoY7cqsjfDw0rQAiFk0V4DSeedZGZjUN_4agv-_MIstlSJz1XRWt-eeGc_HhwPy5j6xtkcA8_EIncKFdhqzcjvJaQEC1inS9bNRIevynZAtnjU0gnT1wxTW8YZEZiVuUhKOuXjPrw-IkaaV8VwgOT8MMFE7yc-2cej7dO4siFCPaM9m-QO_jMQCm98imQT0ImjmBeRfy6_W5WlaViLnWydPfjYKQw9jAA_hotQhoN1Uf81v_npTdYBFuGAOcLMywgfYI8dSCGRecKAPBr1_d8zxHKZsW4P2Qg%3D%3D&attredirects=2) as a range of 2 to 5 SCTs, depending on the lifetime of the certificate.

(Since life times have been limited to 825 days going forward, for new certificates 4 would be the maximum.)

Though the document mentions that EV certificates all need to be CT qualified, the document is defining Chrome's entire CT Qualified definition. This is what Symantec certs are held to, for example. 

Until the policy changes, we should expect that SCTs will range higher than 2 depending on length.

Of course, if we limited our CAs to issuing certs shorter than 15 months, then we could expect to get away with 2 SCTs. ;)